### ISSUE TITLE
What are AllenJS and ArtieJS?

### ISSUE THREAD

**Issue by kevinrobinson** [sonatype-nexus-community/auditjs#193]
I scanned the [changelog](https://github.com/sonatype-nexus-community/auditjs/blob/master/CHANGELOG.md) and noticed this, so since I couldn't understand what it meant I read https://github.com/sonatype-nexus-community/auditjs/commit/592fb93e88b4d4837388ea10bea60223c956cca6.  It seems like the project has added these command line arguments and code as a kind of tribute to particular developers, and that this was done as part of https://github.com/sonatype-nexus-community/auditjs/pull/163, a change that involved 82 files and thousands of lines of changes:

<img width="154" alt="Screen Shot 2020-04-03 at 12 35 31 PM" src="https://user-images.githubusercontent.com/1056957/78384157-aa937480-75a7-11ea-88cd-34bfc1db1ce4.png">

The comment on the PR is "Per @DarthHater "here's 2 months worth of work, YOLO - LOL"

This all seems sort of fun, but along with https://github.com/sonatype-nexus-community/auditjs/issues/192, it makes it very hard to evaluate or trust what has actually changed in the project across a major release.

Thanks for sharing this work and for listening! 👍

**Comment by DarthHater** [comment]
@allenhsieh and @ButterB0wl both worked on the project. They are Easter Eggs for their deep involvement.

Most of our work was done in Alpha/Beta branches, and kinda pushed along the way. 163 was the culmination of everything that made it into alpha/beta along the way.

I can likely do a manual update to the CHANGELOG that brings over the summary of everything from the alpha/beta branches, but I didn't want to do that right off the bat, since the commits are tied to branches that are a bit more "ethereal". 

Basically, what info can I get you that would be helpful!

**Comment by kevinrobinson** [comment]
@DarthHater Got it, thanks!!  Yeah, for me at least if you can help with https://github.com/sonatype-nexus-community/auditjs/issues/192 that would probably be most productive,  :)

**Comment by DarthHater** [comment]
Sounds good!
